### ISSUE TITLE
coordinates api

### ISSUE THREAD

**Issue by malcolmbarrett** [malcolmbarrett/ggdag#20]
I think the coordinates API is unclear, as several people (myself included) have stumbled over it or missed it altogether. I think this needs a few changes:

- [ ] Examples in the new customizing DAGs vignette
- [ ] A separate function that can specify coordinates, something like `dag_coords()` that will overrides `dagitty` object coords and, if it's already tidied, updates the `tidy_dag` coords.

However, I need to think about the second item more

**Comment by vincentarelbundock** [comment]
FWIW, I think the API is very straightforward and natural. But it would be nice to have an example with a data.frame or tribble in the documentation. Personally, I tend to specify coordinates using something like this, since it makes the X-Y coordinates clearer:

```r
library(ggdag)

coords <- read.table(header = TRUE, text = "
  x y name
  0 0 X
  1 0 Z
  2 0 Y
  1 1 U
")

d <- dagify(
  X ~ U,
  Z ~ X,
  Y ~ Z,
  Y ~ U,
  coords = coords)

ggdag(d)
```

**Comment by malcolmbarrett** [comment]
I agree that this way is much clearer and should be more evident in the docs. I think any improvements will center around using the data frame approach

**Comment by go-bayes** [comment]
@vincentarelbundock this is a helpful suggestion. Thank you.
